src/games/domain/games.repository.interface.ts

```
@@ -76,6 +77,11 @@ export interface IGameRepository {
    * @description 사용자가 자주 틀린 카테고리 조회 (오답 비율 포함)
    */
   getFrequentWrongCategories(userId: bigint): Promise<FrequentWrongCategory[]>;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)